src/audio/dai.c

```diff
@@ -462,7 +462,7 @@ static int dai_prepare(struct comp_dev *dev)
 
 	ret = comp_set_state(dev, COMP_TRIGGER_PREPARE);
 	if (ret == COMP_STATE_ALREADY_SET)
-		return ret;
+		return PPL_OP_FURTHER_STOP;
```

Comment:
Been thinking a bit more on this, maybe FURTHER_STOP is not the best word here since the graph walk is recursive and it will continue to walk other paths in the pipeline graph. It's probably better to say we are stopping on this path only i.e.
```
PIPELINE_PATH_STOP
```
That way it should be obvious that the graph walk continues but on other paths.